app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/EventFeedbackFragment.java

```diff
@@ -47,7 +47,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
-        ButterKnife.bind(this, view);
+        if (view != null) {
```

Comment:
I think it's better to let Butterknife throw an exception if View is null in that case. Because if view is null, we cannot continue execution correctly...

Reply:
The exception will be thrown anyway but it does make sense to let the Butterknife initialization throw it